README.md

```diff
@@ -238,6 +238,37 @@ Create a `pump.conf` file:
 
 Settings are the same as for the original `tyk.conf` for redis and for mongoDB.
 
+### Filter Records
+
+This feature adds a new configuration field in each pump called filters and its structure is the following:
+```json
+"filters":{
+  "api_ids":[],
+  "org_ids":[],
+  "response_codes":[],
+  "skip_api_ids":[],
+  "skip_org_ids":[],
+  "skip_response_codes":[]
+}
+```
+The fields api_ids, org_ids and response_codes works as whitelists (APIs and orgs where we want to send the analytics records) and the fields skip_api_ids, skip_org_ids and skip_response_codes works as the opposite (blackslits).
```

Comment:
Could you please change the terms `whitelists` to allow list and and `blacklist` to block list?

Reply:
Done. Remember to update your documentation https://tyk.io/docs/tyk-configuration-reference/tyk-pump-configuration/tyk-pump-configuration/#configuration-1 I have based my description on that

Reply:
thanks I think there's a PR for that :) cc @marksou